templates/CRM/PivotReport/Page/ReportConfig.tpl

```diff
@@ -8,29 +8,29 @@
         <option value="">{ts}-- Select Configuration --{/ts}</option>
         {html_options options=$configList}
       </select>
+      {if ($canManagePivotReportConfig)}
+        <div class="form-item">
+          <input type="button" class="report-config-save-btn btn btn-primary hidden" value="{ts}Save Report{/ts}">
+        </div>
+        <div class="form-item">
+          <input type="button" class="report-config-save-new-btn btn btn-primary" value="{ts}Save As New{/ts}">
+        </div>
+        <div class="form-item">
+          <input type="button" class="report-config-delete-btn btn btn-danger hidden" value="{ts}Delete{/ts}">
+        </div>
+      {/if}
+      <div class="form-item right-align">
```

Comment:
As per @jamienovick 's comment, we should probably move these two divs with the export buttons into #pivot-report-type div.